build.gradle.kts

```diff
@@ -270,6 +270,31 @@ allprojects {
     }
   }
 
+  plugins.withType<SpotlessPlugin>().configureEach {
+    configure<SpotlessExtension> {
+      if (project == rootProject) {
+        kotlin {
+          ktfmt().googleStyle()
+          licenseHeaderFile(rootProject.file("gradle/license-header-java.txt"), "$")
+          target("*.gradle.kts", "buildSrc/*.gradle.kts", "buildSrc/src/**/kotlin/**")
```

Comment:
i am probably missing something but what is `buildSrc` referring to?
should this be `build-tools` ? it seems to be the only folder containing a `kotlin` subfolder in the repo